gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
-version=4.0.2-SNAPSHOT
-ballerinaLangVersion=2201.8.4
+version=4.1.0-SNAPSHOT
```

Comment:
IMO the new version should be `5.0.0` as this is a breaking change for the users

Reply:
In Ballerina libraries we have not followed this approach as I remember. For breaking changes what we have followed up to now is bump the distribution version to the latest Ballerina update version and bump a minor version for the module. 

IMO what you are suggesting is the correct approach, but AFAIR a major version bumps  are not supported yet in the Ballerina dependency resolution logic. @azinneera please correct me if I am wrong.

Reply:
Since this is a breaking change, version updated to `5.0.0` @NipunaRanasinghe @ayeshLK